### PR TITLE
exec command

### PR DESCRIPTION
**Pull request by BSick7**
This PR adds a new CLI command `exec`.
Currently, this is only supported for the aws-fargate provider.
This allows a user to execute commands against running containers in the fargate cluster.
Examples:
```
nullstone exec nullfire-worker dev

# Specify task if there are multiple running
nullstone exec --task=<task-id> nullfire-worker dev
```

### CODE REVIEW

File: aws/ssm/plugin_windows.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package ssm
+
+func getSessionManagerPluginPath() (string, error) {
+	return "session-manager-plugin", nil
```
Comment (ssickles) [inline aws/ssm/plugin_windows.go:4]: If all of these are the same, what's the point of having different ones?

File: cmd/exec.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	"context"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/nullstone.v0/app"
+)
+
+var Exec = func(providers app.Providers) *cli.Command {
+	return &cli.Command{
+		Name:      "exec",
+		Usage:     "Execute command on running service",
```
Comment (ssickles) [inline cmd/exec.go:13]: In the usage description, might want to mention that by not providing a [command], you will connect to a shell session in one of the containers.

File: nullstone/main.go [hunk 3]
```diff
@@ -65,11 +65,12 @@ func main() {
 			cmd.Apps,
 			cmd.Stacks,
 			cmd.Envs,
-			cmd.Deploy(appProviders),
 			cmd.Push(appProviders),
```
Comment (ssickles) [inline nullstone/main.go:68]: I like the ordering here. Order in which someone might use it, clever. :)

**Review by ssickles — APPROVED** [review]
Looks great!

File: aws/ssm/plugin_windows.go (hunk 1 above)
Comment (BSick7) [reply aws/ssm/plugin_windows.go]: Added some logic to increase chances of finding plugin binary.

File: cmd/exec.go (hunk 2 above)
Comment (BSick7) [reply cmd/exec.go]: Done.

**Pull request merged**